doc/api/util.md

```diff
@@ -618,10 +619,41 @@ util.inspect(obj);
 ### util.inspect.custom
 <!-- YAML
 added: v6.6.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/20857
+    description: This is now defined as a shared symbol.
 -->
 
-* {symbol} that can be used to declare custom inspect functions, see
-[Custom inspection functions on Objects][].
+* {symbol} that can be used to declare custom inspect functions.
+
+In addition to being accessible through `util.inspect.custom`, this
+symbol is [registered globally][global symbol registry] and can be
+accessed in any environment as `Symbol.for('util.inspect.custom')`.
+
+```js
+const inspect = Symbol.for('util.inspect.custom');
```

Comment:
While it is important to point this out, I'm not sure if I would recommend encouraging this behavior as it is simply not going to be portable to older versions of Node.js. The change will be transparent to code that uses the existing `util.inspect.custom` accessor. If you're going to include this example, please also include a warning that code written in this way will not be backwards compatible.

Reply:
I can do that, but it won't make much sense without a concrete version e.g.:

Either:

> In addition to being accessible through `util.inspect.custom`, this
symbol is [registered globally](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/for) and — as of 
node v1.2.3 — can be accessed in any environment as 
`Symbol.for('util.inspect.custom')`.

Or:

```js
// Note: this will only work in node v1.2.3 and above
const inspect = Symbol.for('util.inspect.custom');
```

I guess I could add a placeholder, e.g. `REPLACEME`, but that raises the question of why the notice in the `changes` section is insufficient.